.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/rust:1.36.0
+
+    steps:
+      - checkout
+
+      - run:
+          name: setup
+          command: |
+            rustup component add rustfmt --toolchain 1.36.0-x86_64-unknown-linux-gnu
```

Comment:
Why is this version fixed? Normally you fix a version if you plan to support a specific version but do we?

Reply:
My bad. Fixed.